src/main/java/com/jschool/reha/dao/PersonDAOImpl.java

```diff
@@ -18,7 +18,7 @@
     private EntityManager em;
 
     @Override
-    public List<Person> getAllPersons() {
-        return em.createQuery("select p from Person p").getResultList();
+    public List<User> getAllPersons() {
```

Comment:
Rename It to getAllUsers. And move it away from PersonDAOImpl to the UserDAOImpl